src/components/Forms/ReplyForm/ReplyForm.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react';
+import {
+  Container, CssBaseline, TextField, Button, Avatar, Grid,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import cookie from 'react-cookies';
+import { useStyle } from './style';
+import config from '../../../config';
+import { serviceRequest } from '../../../services/serviceRequest';
+import { ErrorMessage } from '../../ErrorMessage';
+
+const path = '/api/v1/post/postreply';
+const domain = config.apiDomain;
+const preparePayload = (method, data) => {
+  const token = cookie.load('jwtToken');
+  const requestHeaders = { Authorization: token };
+  const url = `https://${domain}${path}`;
+  return {
+    method,
+    url,
+    data,
+    headers: requestHeaders,
+  };
+};
+
+export const ReplyForm = (props) => {
+  const classes = useStyle();
+
+  const [content, setContent] = useState('');
+  const [replyForm, setReplyForm] = useState(false);
+  const { commentID } = props;
+  const [isError, setIsError] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
+
+  const avatarURL = 'http://api.adorable.io/avatar/50/';
+
+  const handleChange = (event) => {
+    setContent(event.target.value);
+  };
+
+  const openReplyForm = () => {
+    setReplyForm(true);
+  };
+  const closeReplyForm = () => {
+    setContent('');
+    setReplyForm(false);
+  };
+
+  const handleSubmit = async (event) => {
```

Comment:
You should `trim()` all user input before they submit a request to backend API so strings with many spaces would not be submitted as successful request.

Reply:
I check and trim it in the reply button in the disabled attribute

Reply:
Thank you for pointing that out! I will resolve this conversation.

Reply:
Thank you for pointing it out! I will resolve this conversation.